tests/test_extrusions.py

```diff
@@ -0,0 +1,66 @@
+from plugins.extrusions import extrusions
+
+
```

Comment:
You might want to do some stuff with [BoundingBox](https://github.com/CadQuery/cadquery/blob/ba1dfe448f47ea31e083865bc3e6be19982677f1/cadquery/occ_impl/shapes.py#L560) here to check that the length and centering arguments are doing what you expect.